Userland/Applications/Piano/PlayerWidget.cpp

```diff
@@ -11,18 +11,55 @@
 #include "TrackManager.h"
 #include <LibGUI/BoxLayout.h>
 #include <LibGUI/Button.h>
+#include <LibGUI/ComboBox.h>
+#include <LibGUI/ItemListModel.h>
+#include <LibGUI/Label.h>
 
 PlayerWidget::PlayerWidget(TrackManager& manager, AudioPlayerLoop& loop)
     : m_track_manager(manager)
     , m_audio_loop(loop)
 {
     set_layout<GUI::HorizontalBoxLayout>();
     set_fill_with_background_color(true);
+    m_track_number_choices.append("1");
 
     m_play_icon = Gfx::Bitmap::try_load_from_file("/res/icons/16x16/play.png").release_value_but_fixme_should_propagate_errors();
     m_pause_icon = Gfx::Bitmap::try_load_from_file("/res/icons/16x16/pause.png").release_value_but_fixme_should_propagate_errors();
     m_back_icon = Gfx::Bitmap::try_load_from_file("/res/icons/16x16/go-back.png").release_value_but_fixme_should_propagate_errors();    // Go back a note
     m_next_icon = Gfx::Bitmap::try_load_from_file("/res/icons/16x16/go-forward.png").release_value_but_fixme_should_propagate_errors(); // Advance a note
+    m_add_track_icon = Gfx::Bitmap::try_load_from_file("/res/icons/16x16/plus.png").release_value_but_fixme_should_propagate_errors();
+    m_next_track_icon = Gfx::Bitmap::try_load_from_file("/res/icons/16x16/go-last.png").release_value_but_fixme_should_propagate_errors();
+
+    RefPtr<GUI::Label> label = add<GUI::Label>("Track ");
```

Comment:
There shouldn't be a space in a label. If the spacing is incorrect without it, add margins to the label.

Reply:
Updated